orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy

```diff
@@ -47,6 +48,9 @@ class BakeStage implements StageDefinitionBuilder {
   @Autowired
   RegionCollector regionCollector
 
+  @Autowired
+  Clock clock = systemUTC()
```

Comment:
The test for this (that was using Spock's `Spy`) started failing so I figured it would make sense to adopt a better pattern anyway.